### PR TITLE
hotfix/IISCRUM-1700

### PR DESCRIPTION
**Pull request by timyllym**
- Check for tab row child panels when adding new panel
- Get new id's for tab row panel child panels

### CODE REVIEW

**Review by sapajune — APPROVED** [review]
Looks ok.

Kind of hard to follow the numbering logic.

**Comment by timyllym** [comment]
When new panel is added, it goes through all existing panels in dashboard and checks for the highest id number. It also loops through all child panels that root panels possibly have. New panel id number is the highest existing number plus 1.

**Comment by sapajune** [comment]
Yeah. I got that but took awhile.
